cf-testd/cf-testd.c

```diff
@@ -215,7 +220,54 @@ bool CFTestD_GetServerQuery(ServerConnectionState *conn, char *recvbuffer)
         return false;
     }
 
-    return CFTestD_ReturnQueryData(conn, query);
+    if (config->report_len == 0)
+    {
+        Log(LOG_LEVEL_INFO,
+            "No report file argument so returning canned data from enterprise plugin server\n");
+        return CFTestD_ReturnQueryData(conn, query);
+    }
+
+    Log(LOG_LEVEL_INFO,
+        "Report file argument specified. Returning report of length %d",
+        config->report_len);
+
+    char *header;
+
+    // TODO for variables reports we will eventually need to split on \m
+    // instead of \n since variable values can contain '\n's.
+
+    size_t num_items =
+        StringCountTokens(config->report, config->report_len, "\n");
+
+    if (num_items < 3)
+    {
+        Log(LOG_LEVEL_ERR,
+            "report file must have a CFR timestamp header line and at least two lines of items\n");
+        return false;
+    }
+
+    StringRef ts_ref =
+        StringGetToken(config->report, config->report_len, 0, "\n");
+    char *ts = xstrndup(ts_ref.data, ts_ref.len);
+    header   = StringFormat("CFR: 0 %s %d\n", ts, config->report_len);
+    SendTransaction(
+        conn->conn_info, header, SafeStringLength(header), CF_MORE);
```

Comment:
This block looks to me like it got scattered... (I know it's "clang-format-great")

Reply:
What do you mean scattered? Let's have a discussion here: https://github.com/cfengine/core/pull/3045

Reply:
that one is only two chars over 78, I'd say put it back to a single line.

```
    SendTransaction(conn->conn_info, header, SafeStringLength(header), CF_MORE);
```

But like you said @olehermanse discuss over in #3045 and maybe I should just revert all these clang-format related changes and stick with plain and simple, minimal changes to conform to things like curly brackets and such.